examples/meshes.ipynb

```diff
@@ -0,0 +1,317 @@
+    "\n",
+    "The function `find_mesh` can be used to find a specific mesh (filtering by name) or, if run without arguments, it will list all the available meshes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0145871e-8e8b-4542-b2f0-870aad6de379",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['cube40_equi_grains8_gsize20',\n",
+       " 'cube80_equi_grains8_gsize40',\n",
+       " 'cube160_equi_grains8_gsize80',\n",
+       " 'cube40_colu_grains8_gsize20',\n",
+       " 'cube80_colu_grains8_gsize40',\n",
+       " 'cube160_colu_grains8_gsize80',\n",
+       " 'cube40_3plat_grains12_gsize20',\n",
+       " 'cube80_3plat_grains12_gsize40',\n",
+       " 'cube160_3plat_grains12_gsize80']"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "find_mesh()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc73de1e-df89-44dc-9aca-78a1cbb106be",
+   "metadata": {},
+   "source": [
+    "These meshes are stored on Zenodo: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.16736878.svg)](https://doi.org/10.5281/zenodo.16736878)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "94085c5c-7e3e-4fec-ad37-d80811db44ac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mesh = Mesh(\"cube40_3plat_grains12_gsize20\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "bc8a279c-dfd2-4bd9-b6ed-198992489c30",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Mesh(cube40_3plat_grains12_gsize20)"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mesh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7689933f-58af-40b8-9d78-895531d74d6f",
+   "metadata": {},
+   "source": [
+    "We can see some information in the `__str__` dunder method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "4c496d0e-5325-4a10-8eee-5112aa87c4b3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "name:\n",
+      "\tcube40_3plat_grains12_gsize20\n",
+      "description:\n",
+      "\tCube of side length 40 with 12 grains of size 20 generated as regular parallelepipeda on three layers.\n",
+      "geometry:\n",
+      "\tcube\n",
+      "length:\n",
+      "\t40\n",
+      "ngrains:\n",
+      "\t8\n",
+      "grain_size:\n",
+      "\t20\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(mesh)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b13e874-a44d-4d3b-b402-8a41b2426f0f",
+   "metadata": {},
+   "source": [
+    "## Generate the mesh\n",
+    "\n",
+    "We can generate the mesh in a specific local destination using the `write` method:"
```

Comment:
Please rephrase, I don't like the word "generate". It sounds too much like we would actually mesh something but we are just downloading/copying a file.